File: Suma de dos numeros en Form VB .NET Framework/Suma de dos numeros en Form VB .NET Framework/Form1.vb
Public Class Form1
    Private Sub btnSumar_Click(sender As Object, e As EventArgs) Handles btnSumar.Click
        Try
            Dim numero1 As Double = Double.Parse(txtNumero1.Text)
            Dim numero2 As Double = Double.Parse(txtNumero2.Text)

            Dim resultado As Double = numero1 + numero2

            txtResultado.Text = resultado.ToString()
        Catch ex As FormatException
            MessageBox.Show("Por favor, ingresa números válidos en los campos 1 y 2.", "Error", MessageBoxButtons.OK, MessageBoxIcon.Error)
        End Try
    End Sub
End Class

File: Suma de dos numeros Form en c# .NET/Suma de dos numeros en c# .NET/Form1.cs
namespace Suma_de_dos_numeros_en_c__.NET
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnSumar_Click(object sender, EventArgs e)
        {
            try
            {
                double numero1 = double.Parse(txtNumero1.Text);
                double numero2 = double.Parse(txtNumero2.Text);

                double resultado = numero1 + numero2;

                txtResultado.Text = resultado.ToString();
            }
            catch (FormatException)
            {
                MessageBox.Show("Por favor, ingresa números válidos en los campos 1 y 2.", "Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
            }
        }
    }
}